### DISCUSSION TITLE
[Feature Request] No Splitting Snaps

### DISCUSSION THREAD

**Discussion by ghost** [rodit/SnapMod#93] [Ideas & Feature Suggestions]
With api it's possible by pushing the full file. Should be similar to app just hook and remove the splitting.

PS: No one dm me for api lmao

**Comment by Magna-Graecia** [comment]
Do you mean sending video snaps that are longer than 10 seconds without having them be split into multiple snaps? We can already do this with SnapMod. If you go in a chat, open the gallery drawer, check the video you want to send, and hit send, it will get sent as one full snap without being split up at all. I've done this with videos up to a minute long, and they get sent as one minute-long snap. The only downside to this is that you can't add any edits to the video (text, filters, etc) since you're sending directly and not loading it into the camera window.

If you instead go to a chat, open the gallery drawer, and long press on a video, it will get loaded into the camera window where you can add edits and such. However, doing it this way will cause the video to get split into multiple snaps if it's longer than 10 seconds.

**Comment by ghost** [comment]
@Magna-Graecia no i mean stories / direct snaps its possible to not split at all.

**Comment by Magna-Graecia** [comment]
> @Magna-Graecia no i mean stories / direct snaps its possible to not split at all.

You mean when saving them?

**Comment by RSR1337** [comment]
> > @Magna-Graecia no i mean stories / direct snaps its possible to not split at all.
> 
> You mean when saving them?

He mean , upload more than 1min snap without split to the stories or Creator public profile . 

@rodit  any plans to add this feature?

**Comment by rodit** [comment]
I probably will at some point, yes.

**Comment by Magna-Graecia** [comment]
@iRashiD-AE You can't upload a video longer than 10 seconds to stories/public profile without it being split into multiple snaps, but his reply to me stated direct snaps as well. I just tested right now with a video that is 1 minute and 5 seconds long, and I can definitely send it as a direct live snap with no splits to someone simply by opening the chat with them, opening the gallery drawer, tapping on the video to checkmark it, and pressing send.

Normally you can't even send a 60+ long snap to someone/story period, splits or no splits, so the fact that we can using SnapMod is cool as it is. If you open a chat, open the gallery drawer, long press on a video longer than 60 seconds, you will see that it gets loaded into the camera window as long as "Bypass Video Length Restrictions" is enabled, and then you can send to someone/story from there. Albeit with splits.

If you open memories and try to send the same 60+ second long video from there, it will say "Videos longer than 60 seconds can't be sent", and 10+ second long videos sent from memories would be split anyway even if you could send it.

**Comment by ghost** [comment]
@Magna-Graecia lmao yes u can i have api access i would know i posted hour long stuff on my story no splits

**Comment by Magna-Graecia** [comment]
> @Magna-Graecia lmao yes u can i have api access i would know i posted hour long stuff on my story no splits

I'm not saying it isn't possible. I was only stating that we can't do it with SnapMod as of right now, but we can already send unsplit live direct snaps to people. Your reply to me said "direct snaps", so I was replying to that.

Either way, rodit has stated that he plans to work on it.

**Comment by RSR1337** [comment]
@rodit When will you add the no split feature?

**Reply by rodit** [reply]
Not sure

**Comment by rodit** [comment]
Added in 1.6.3
